src/java/Main.java

```diff
@@ -0,0 +1,49 @@
+package java;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Civil AN=new Civil();
+        AN.BusinessClass=false;
+        AN.NumberOfPassengers=20;
+        AN.maksSpeed=700;
+        AN.MinLength=250;
+        AN.power=1000;
+        AN.WingSpan=50;
+        AN.Vyvod();
+        AN.Power();
+        AN.loading();
+        Military MIG= new Military();
+        MIG.maksSpeed=600;
+        MIG.MinLength=200;
+        MIG.power=1500;
+        MIG.EjectionSystems=true;
+        MIG.NumberOfMissiles=10;
+        MIG.Vyvod();
+        MIG.Power();
+        MIG.ejection();
+        MIG.shot();
+        Light_vehicle BMW=new Light_vehicle();
```

Comment:
@Hlieb имена объектов/экземпляров класса всегда с маленькой буквы
```suggestion
        Light_vehicle bMW=new Light_vehicle();
```